ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/GeoCityLookup.java

```diff
@@ -173,47 +257,10 @@ private void loadResourcesOnFirstMessage() throws IOException {
       if (geoCityDatabase == null || !geoCityDatabase.isAccessible()) {
         throw new IllegalArgumentException("--geoCityDatabase must be defined for GeoCityLookup!");
       }
-      loadGeoIp2City();
+      geoIP2City = GeoCityLookup.getOrCreateSingletonGeoCityReader(geoCityDatabase);
```

Comment:
Pretty sure the class name isn't necessary here:
```suggestion
      geoIP2City = getOrCreateSingletonGeoCityReader(geoCityDatabase);
```

Reply:
> Pretty sure the class name isn't necessary here:

You are correct. I intentionally included the class name to be explicit about it being a static method, but I can remove if that seems overly verbose. Having `singleton` in the name is probably sufficient to alert future devs to what's going on.